src/components/MyKiva/GoalSetting/GoalSelector.vue

```diff
@@ -241,23 +249,44 @@ const handleContinue = () => {
 	}
 };
 
-onMounted(() => {
-	if (womenLoansLastYear.value > SAME_AS_LAST_YEAR_LIMIT) {
-		const growALittleOption = Math.ceil(womenLoansLastYear.value * 1.25);
+async function loadWomenLoansThisYear() {
+	const currentYear = new Date().getFullYear();
+	const categoryLoansThisYear = await getCategoryLoansByYear(currentYear);
+	womenLoansThisYear.value = categoryLoansThisYear?.find(
+		categoryLoans => categoryLoans.id === ID_WOMENS_EQUALITY
```

Comment:
This filtering logic is currently implemented in two locations, but it could be simplified by an extra parameter to getCategoryLoansByYear to return the filtered results.